tests/unit/Actions.spec.ts

```diff
@@ -256,37 +242,37 @@ describe.skip('Emits error event if NR validation fails in file and pay', () =>
       }))
 
     // init store
-    store.state.stateModel.tombstone.currentDate = '2020/01/29'
-    store.state.stateModel.nameRequest = {
+    store.stateModel.tombstone.currentDate = '2020/01/29'
+    store.stateModel.nameRequest = {
       entityType: 'BEN',
       nrNumber: 'NR 1234567',
       details: { approvedName: 'My Name Request Inc.' }
-    }
-    store.state.stateModel.tombstone = {
+    } as any
+    store.stateModel.tombstone = {
       keycloakRoles: [],
       authRoles: [],
       userEmail: 'completing-party@example.com'
-    }
-    store.state.stateModel.certifyState = {
+    } as any
+    store.stateModel.certifyState = {
       valid: true,
       certifiedBy: 'Some certifier'
     }
-    store.state.stateModel.tombstone.entityType = 'BEN'
-    store.state.stateModel.validationFlags = {
+    store.stateModel.tombstone.entityType = CorpTypeCd.BENEFIT_COMPANY
+    store.stateModel.validationFlags = {
       flagsCompanyInfo: {
         isValidOrgPersons: true,
         isValidShareStructure: true
       },
       flagsReviewCertify: {
         isValidEffectiveDate: true
       }
-    }
+    } as any
```

Comment:
as any aren't ideal.. we should probably fix these. But they'll need to have the required fields.

Reply:
I think we shouldn't use types in the unit tests.

Reply:
Is the new linter (or Typescript) complaining about undeclared properties?

Reply:
Yup

Reply:
OK, that's new and nice 👍